### ISSUE TITLE
Floating menu not responsive for videos, images and stickers

### ISSUE THREAD

**Issue by felipebochehin87** [swissspidy/web-stories-wp#11258]
## Bug Description

The floating menu not responsive for videos, images and stickers. Once user hits Control + Alt + P, the cursor gets "stuck" on opacity field and using tab or arrow buttons on the keyboard take no effect.

## Expected Behaviour

User should be able to navigate in the floating menu using Tab and the arrow keys (left and right).

## Steps to Reproduce

1. Add a video, an image and a sticker the canvas
2. Use the new shortcut Control + Alt + P
3. User Tab button and also the arrow keys - note that the focus is stuck on the opacity field

## Screenshots

https://user-images.githubusercontent.com/841956/163134779-358467ed-6cdb-4d2a-add9-08c29faf3183.mp4

## Additional Context

- Plugin Version: 1.20.0-alpha
- WordPress Version: 5.9
- Operating System:
- Browser:

**Comment by swissspidy** [comment]
Introduced in #11223 it seems

cc @littlemilkstudio 

Aside: this new keyboard shortcut should be documented in the keyboard shortcuts modal.

**Comment by BrittanyIRL** [comment]
This is getting fixed here: https://github.com/GoogleForCreators/web-stories-wp/pull/11273 

And the shortcut was added here: https://github.com/GoogleForCreators/web-stories-wp/pull/11243

I'll validate this ticket once 11273 is merged and then close it.

**Comment by BrittanyIRL** [comment]
Resolves with #11162

**Comment by felipebochehin87** [comment]
Same as https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/googleforcreators/web-stories-wp/11162
